aws-cs-lambda/pulumi/Program.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Pulumi;
+using Pulumi.Aws.Lambda;
+using Pulumi.Aws.Iam;
+
+class LambdaUtil
+{
+    public static Pulumi.AssetArchive buildArchive(string lambdaArchivePath)
+    {
+        var immutableDictBuilder = System.Collections.Immutable.ImmutableDictionary.CreateBuilder<string, AssetOrArchive>();
+        immutableDictBuilder.Add(".", new FileArchive(lambdaArchivePath));
+        return new Pulumi.AssetArchive(immutableDictBuilder.ToImmutable());
```

Comment:
i'll make htis better so you don't need the ImmutableDict stuff either.

Reply:
Am I correct that `AssetArchive` is not needed at all in this particular case? Just pass `FileArchive` to `Code`. Or is there a difference?

Reply:
Was able to update this as suggested by @mikhailshilkov